sw/airborne/firmwares/rotorcraft/autopilot_firmware.c

```diff
@@ -149,10 +150,10 @@ static void send_fp_min(struct transport_tx *trans, struct link_device *dev)
   uint16_t gspeed = stateGetHorizontalSpeedNorm_f() / 100;
 #endif
   pprz_msg_send_ROTORCRAFT_FP_MIN(trans, dev, AC_ID,
-                              &(stateGetPositionEnu_i()->x),
-                              &(stateGetPositionEnu_i()->y),
-                              &(stateGetPositionEnu_i()->z),
-                              &gspeed);
+                                  &(stateGetPositionEnu_i()->x),
+                                  &(stateGetPositionEnu_i()->y),
+                                  &(stateGetPositionEnu_i()->z),
+                                  &gspeed);
```

Comment:
Is this respacing required?

Reply:
It's what the autoscript did, so I don't really care. But if I change it back we probably have this issue again in my next PR ;)